vaadin-element.html

```diff
@@ -29,6 +29,10 @@
   </template>
 
   <script>
+    if (!Polymer.Element) {
+      throw 'Wrong Polymer version, expected Polymer v2.0.0 or later.';
```

Comment:
`throw` should throw an `Error` object, not string. http://eslint.org/docs/rules/no-throw-literal